app/assets/stylesheets/partials/_contact-form.scss

```diff
@@ -0,0 +1,60 @@
+form#contact-form {
+  width: 100%;
+
+  label {
+    color: #666;
+    font-size: 15px;
+    font-weight: bold;
+    margin-bottom: 8px;
+  }
+
+  textarea, 
+  input[type="text"] {
+    font-size: 13px;
+    margin-bottom: 8px;
+    width: 100%;
+  }
+
+  textarea {
+    height: 100px;
+  }
+
+  input[type="submit"] {
+    background-color: #0069A0;
+    border-radius: 5px;
+    cursor: pointer;
+    display: block;
+    font-size: 13px;
+    margin: 0 auto;
+    width: 140px;
+  }
+
+  hr {
+    margin: 15px 0;
+  }
+}
+
+div#contact-alert {
+  display: none;
+}
+
+.alert-box {
+  border-radius: 4px;
+  font-size: 14px;
+  text-align: center;
```

Comment:
Properties should be ordered border-radius, font-size, margin-top, padding, text-align, width